src/Maestro/tests/DependencyUpdateErrorProcessor.Tests/UpdateGithubIssueTest.cs

```diff
@@ -119,5 +120,171 @@ public Issue GetIssue()
             );
             return issue;
         }
+
+        [Fact]
+        public async Task CreateIssueAndAddAdditionalComment()
+        {
+            RepositoryBranchUpdateHistoryEntry firstError =
+                new RepositoryBranchUpdateHistoryEntry
+                {
+                    Repository = RepoUrl,
+                    Branch = Branch,
+                    Method = MethodName,
+                    Timestamp = new DateTime(2200, 1, 1),
+                    Arguments = $"[\"{SubscriptionId}\",\"{MethodName}\",\"{ErrorMessage}\"]",
+                    Success = false,
+                    ErrorMessage = ErrorMessage,
+                    Action = "Creating new issue"
+                };
+            RepositoryBranchUpdateHistoryEntry secondError =
+                new RepositoryBranchUpdateHistoryEntry
+                {
+                    Repository = RepoUrl,
+                    Branch = Branch,
+                    Method = "ProcessPendingUpdatesAsync",
+                    Timestamp = new DateTime(2200, 2, 1),
+                    Arguments = "ProcessPendingUpdatesAsync error",
+                    Success = false,
+                    ErrorMessage = ErrorMessage,
+                    Action = "Create a new issue comment",
+                };
+
+            Context.RepoBranchUpdateInMemory = new List<RepositoryBranchUpdateHistoryEntry>
+                {firstError , secondError};
+            Mock<Maestro.Data.Models.Subscription> subscription = new Mock<Maestro.Data.Models.Subscription>();
```

Comment:
This can just be
```
new Subscription {
  Id = Guid.Parse(SubscriptionId),
  SourceRepository = "Source Repo",
  TargetRepository = "Target Repo",
}
```

There is no reason to make a mock. This is just a data object.

Reply:
I changed that in createGithubIssue file and missed it here. Changing it.